.github/workflows/tests_without_binary.yml

```
@@ -103,6 +103,8 @@ jobs:
           SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" python -m pip install . -vv
 
       - name: Install backend device
+        # FIXME: Remove this condition with v0.37.0 release
+        if: matrix.pl_backend != 'lightning_tensor' && inputs.lightning-version != 'stable'
```

Shouldn't this be an `or`? 
```suggestion
        if: matrix.pl_backend != 'lightning_tensor' || inputs.lightning-version != 'stable'
```

---

No, this is supposed to trigger for
1. `pl_backend=lightning_tensor` and `lightning-version=latest`
2. other lightning simulators and `lightning-version=stable`

This is going to be skipped for only when `pl_backend=lightning_tensor` and `lightning-version=stable`.

---

Say I have `'lightning_qubit' != 'lightning_tensor' && 'stable' != 'stable'` which is `True && False` then it won't run this section, right? So this will not install LQ with `stable`, is that what we want?

---

Oops! My bad -- I thought it's already `or` and you're suggesting to change it! (never read comments while having lunch :smile: )